integration/localnet/Makefile

```diff
@@ -1,5 +1,5 @@
-COLLECTION = 3
-CONSENSUS = 3
+COLLECTION = 2
+CONSENSUS = 1
```

Comment:
Not sure that we want to change these defaults for everyone. You can initialize localnet with different node counts with eg. `make init -e COLLECTION=2 -e CONSENSUS=1`